.python-version

```diff
@@ -0,0 +1 @@
+pystac-dev
```

Comment:
This file seems like a `pipenv`-specific file that might not belong in source control (since we're not officially a `pipenv` project?)

Reply:
Yeah, this is a `pyenv-virtualenv` thing. I'll add it to the `.gitignore`.

Reply:
Fixed this in the latest push